.changeset/forty-eggs-raise.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/partytown': patch
+---
+
+Allows overriding the `lib` option
```

Comment:
```suggestion
Adds the ability to override the `lib` option in `astro.config.mjs`
```
What do you think about something like this? I like priming the reader with "Adds" at the beginning of the sentence to let them know there's something new they can now do. I also like the tiny extra bit of description here. Take from this what you will! The original was fine, I just think it can be even more helpful! :raised_hands: